polaris-react/src/components/Filters/Filters.scss

```diff
@@ -154,34 +162,6 @@
       width: var(--p-space-4);
     }
   }
-
-  #{$se23} & {
-    background: var(--p-color-bg-secondary-experimental);
-    border-radius: var(--p-border-radius-2);
-    border: var(--p-color-border) dashed var(--p-border-width-1);
-
-    &:hover {
-      background: transparent;
-      border-style: solid;
-    }
-
-    &:focus {
-      background: transparent;
-      outline-offset: var(--p-border-width-2);
-    }
-
-    &:active {
-      background: var(--p-color-bg-subdued);
-
-      #{$se23} & {
-        background: var(--p-color-bg-secondary-experimental);
```

Comment:
Looks like a remnant from the  subdued > secondary token migration. 
On one hand, I think the intention here is to apply `secondary-experimental` instead of `subdued`.
On the other hand, the active background color is noticeably different to the default background color feels good (even if it is a bug) 

Not a designer though, so defering to @sarahill  or @bernardoamc  to weigh in here.